config/initializers/0_clowder-config.rb

```diff
@@ -30,7 +30,7 @@
   if kafka_security_protocol
     if kafka_security_protocol == 'sasl'
       kafka_server_config[:security_protocol] = 'sasl_ssl'
-      kafkaCaFile = Tempfile.create(mode: 'wt')
+      kafkaCaFile = Tempfile.create(mode: File::WRONLY)
```

Comment:
It might need `CREAT` and `EXCL`.
```
File::WRONLY|File::CREAT|File::EXCL
```
But I guess it might be left as default (with read/write).

Reply:
No https://github.com/ruby/ruby/blob/master/lib/tempfile.rb#L352